packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -0,0 +1,41 @@
+@use '@tokens' as tokens;
+
+$spacing: tokens.$space-300;
```

Comment:
Maybe we could improve it to `gap`? It provides a better clue what it does rather than`spacing`… Just in case it is actually meant for spacing, of course.